src/ImageCarouselReact/widget/components/ImageCarousel.tsx

```diff
@@ -181,17 +173,16 @@ export class ImageCarousel extends React.Component<ImageCarouselProps, {}> {
         );
     }
     /**
-     * Handles the onlick for carousel items
+     * Handles the onclick for carousel items
      */
-    private onItemClick(onClickProps?: IOnclickProps) {
+    private onItemClick(onClickProps?: OnclickProps) {
         logger.debug(this.props.widgetId + ".onItemClick");
-        if (onClickProps.onClickEvent === "microflow" && onClickProps.clickMF) {
+        if (onClickProps.onClickEvent === "callMicroflow" && onClickProps.clickMF) {
             this.clickMicroflow(onClickProps.clickMF, onClickProps.guid);
-        } else if (["content", "popup", "modal"].indexOf(onClickProps.onClickEvent) > -1 &&
-            onClickProps.page) {
+        } else if (onClickProps.onClickEvent === "openPage" && onClickProps.page) {
             this.showPage({
                 guid: onClickProps.guid,
-                location: onClickProps.onClickEvent,
+                pageLocation: onClickProps.pageLocation,
                 pageName: onClickProps.page,
             });
         } else {
```

Comment:
line 213: Params could be declare literal in the call of the action.
            `
params: {
                applyto: guid ? "selection" : "none",
                guids: guid ? [guid] : undefined,
            },
`